src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -495,3 +495,29 @@
     text: az backup vault show --name MyRecoveryServicesVault --resource-group MyResourceGroup
     crafted: true
 """
+helps['backup vault update'] = """
+type: command
+short-summary: Updates an existing Recovery Services vault.
```

Comment:
Please use the first person voice
```suggestion
short-summary: Update an existing Recovery Services vault.
```